config/testing.js

```diff
@@ -3,7 +3,11 @@ exports.config = {
   isTesting: true,
   common: {
     database: {
-      name: process.env.NODE_API_DB_NAME_TEST
+      host: process.env.DB_HOST_TEST,
```

Comment:
why are you changing this?

Reply:
because we dont use URL anymore for sequelize

Reply:
This might be the reason why the CI is failing. Postgres will expose `DB_HOST` and you'll need that to connect